core/src/main/java/com/google/zxing/maxicode/decoder/DecodedBitStreamParser.java

```diff
@@ -100,7 +100,7 @@ static DecoderResult decode(byte[] bytes, int mode) {
         result.append(getMessage(bytes, 1, 77));
         break;
     }
-    return new DecoderResult(bytes, result.toString(), null, String.valueOf(mode));
+    return new DecoderResult(bytes, result.toString(), null, String.valueOf(mode), 0);
```

Comment:
If 0 is a default value, how about just overloading the constructor with a version that doesn't take this input?

Reply:
Sure, I'll add that. 
I actually would have used a parameter default if Java had support for that. Overloading is a bit of a pain.

Reply:
You can leave this as is or just add another overload, whatever makes more sense to you

Reply:
Both variants are fine to me. 
As you leave it to me, I'll opt for keeping it as it is. Feels clearer to me where to modify the default in the BitStreamParsers in the case of adding a more detailed detection in the future.

Reply:
OK can this use the same constructor now rather than add the default value?